practice-app/website/api/home.py

```diff
@@ -12,12 +13,13 @@ def home_route():
     """
     query = request.args.get("query", None)
     if query:
-        events, art_items = get_content_with_filter(query)
+        events, art_items, copyright_reports = get_content_with_filter(query)
```

Comment:
**Please note:** 

If you decide to restrict the visibility of copyright reports from the guest users, you should not show those in the search results, too.